tasks/elasticsearch-template.yml

```diff
@@ -15,15 +15,30 @@
 
 - name: Install templates
   uri:
-    url: "http://{{es_api_host}}:{{es_api_port}}/_template/{{item | filename}}"
+    url: "{{es_api_uri}}/_template/{{item | filename}}"
     method: PUT
     status_code: 200
     user: "{{es_api_basic_auth_username | default(omit)}}"
     password: "{{es_api_basic_auth_password | default(omit)}}"
     force_basic_auth: yes
     body_format: json
     body: "{{ lookup('file', item) }}"
-  when: load_templates.changed and es_start_service
+  when: load_templates.changed and es_start_service and not es_enable_xpack or not es_xpack_features is defined or "security" not in es_xpack_features
+  with_fileglob:
+    - "{{ es_templates_fileglob | default('') }}"
+  run_once: True
+
+- name: Install templates with auth
```

Comment:
I think this changed since you started working on this. The above task was refactored to work for auth and no auth by doing `user: "{{es_api_basic_auth_username | default(omit)}}"`

Reply:
https://github.com/elastic/ansible-elasticsearch/commit/a677218fa85a5cbbc7e835a3ede8e5a6c7cee2bf#diff-8cdc0121d463a439bd46901832ee0c2f is where it was changed, this commit seems to revert it.

Reply:
Nope this was a `rebase` mistake, I think that https://github.com/elastic/ansible-elasticsearch/pull/479/commits/4fc4c6d0bef174a26d209770fbf79cfcad0d1f92 fixes it